spec/index.bs

```diff
@@ -7121,11 +7121,17 @@ dictionary GPURenderPassDescriptor : GPUObjectDescriptorBase {
         The set of {{GPURenderPassColorAttachment}} values in this sequence defines which
         color attachments will be output to when executing this render pass.
 
+        Due to [=compatible usage list|usage compatibility=], no color attachment
+        may alias another attachment or any resource used inside the render pass.
+
     : <dfn>depthStencilAttachment</dfn>
     ::
         The {{GPURenderPassDepthStencilAttachment}} value that defines the depth/stencil
         attachment that will be output to and tested against when executing this render pass.
 
+        Due to [=compatible usage list|usage compatibility=], no writable depth/stencil attachment
```

Comment:
it's not possible to have multiple depth attachments in a pass, anyway, so this sentence could be simplified

Reply:
It's meant to be unambiguous - we can't currently use textures of the same format for both color and depth outputs, but we could if red/depth could be reinterpreted by createView. The reader of this sentence shouldn't need to know all the context.